client/signup/steps/plans/index.jsx

```diff
@@ -46,16 +47,24 @@ class PlansStep extends Component {
 			} );
 		}
 
-		SignupActions.submitSignupStep( {
+		const step = {
 			processingMessage: isEmpty( cartItem )
 				? translate( 'Free plan selected' )
 				: translate( 'Adding your plan' ),
 			stepName,
 			stepSectionName,
 			cartItem,
-			privacyItem,
 			...additionalStepData
-		}, [], { cartItem, privacyItem } );
+		};
+
+		const providedDependencies = { cartItem };
+
+		if ( privacyItem ) {
```

Comment:
For my own understanding: why is this change needed? Shouldn't a loosely false `privacyItem` be ignored [here](https://github.com/Automattic/wp-calypso/blob/master/client/lib/signup/step-actions.js#L264)?